### DISCUSSION TITLE
Can't make my selection!

### DISCUSSION THREAD

**Discussion by ghost** [LSPosed/MagiskOnWSALocal#271] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/79799243/205424007-4f92b162-8d7f-4fc8-bff7-f6d09d45a627.png)
I am using a virtual box to run the ubuntu 
Even using the arrows marks I can't make the selection 
The asterisk symbol stays in the default value 
Anything I am missing?

**Comment by Howard20181** [comment, marked as answer]
Press the space bar
